examples/nextjs/.env.local.example

```diff
@@ -1,2 +1,4 @@
 DOTCMS_AUTH_TOKEN=
-NEXT_PUBLIC_DOTCMS_HOST=http://localhost:8080
\ No newline at end of file
+NEXT_PUBLIC_DOTCMS_HOST=http://localhost:8080
+NEXT_PUBLIC_EXPERIMENTS_API_KEY=analytic-api-key-from-dotcms-portlet
+NEXT_PUBLIC_EXPERIMENTS_DEBUG=true
```

Comment:
FYI, these env vars are not showing in client (runtime) when is deployed to Google Cloud. We're not doing anything, but just want to let you know.

Reply:
I will take into consideration in the documentation of this.